dbschema/1.tables.sql

```diff
@@ -135,3 +135,25 @@ CREATE INDEX tbl_token_multi_ownerships_owner_address ON tbl_token_multi_ownersh
 CREATE INDEX tbl_token_multi_ownerships_latest_transfer_date ON tbl_token_multi_ownerships (latest_transfer_date);
 CREATE INDEX tbl_token_multi_ownerships_latest_transfer_value ON tbl_token_multi_ownerships (latest_transfer_value);
 CREATE INDEX tbl_token_multi_ownerships_latest_transfer_transaction_hash ON tbl_token_multi_ownerships (latest_transfer_transaction_hash);
+
+CREATE TABLE tbl_collection_hourly_activity (
+    id BIGSERIAL PRIMARY KEY,
+    created_date TIMESTAMP WITHOUT TIME ZONE NOT NULL,
+    updated_date TIMESTAMP WITHOUT TIME ZONE NOT NULL,
+    address TEXT NOT NULL,
+    date TIMESTAMP WITHOUT TIME ZONE NOT NULL,
```

Comment:
unique on date + address pls

Reply:
can you see if we can enforce that the date only has day + hour, i.e. no minutes and seconds and milliseconds

Reply:
I checked and didn't see any straightforward means of accomplishing that